ENTO-ADULT SURVEILLANCE/.module.json

```diff
@@ -0,0 +1,9 @@
+{
+    "id": "ZNcMvLVhA2e",
+    "name": "ENTO-ADULT SURVEILLANCE",
+    "instance": "http://localhost:8080",
```

Comment:
instance localhost:8080? Let's say that this instance is running in 
http://localhost:8083 but that's being serve outside via https://extranet.who.int/dhis2-ento-vc

Would this instance URL still be the same? I'm a bit confused with this field as the module might not be related to an instance...right?

Reply:
ok, understood via Slack that it's only an info field not used in any other place so...changing it into the real instance and merging it